README.adoc

```diff
@@ -13,14 +13,19 @@ image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]
 :note-caption: :information_source:
 NOTE: This simple example ignores https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-cache[analysis cache] and https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#parallel-code-scan[multithreaded execution] both of which can boost analysis significantly.
 
-It is very easy to analyze a C, C++ and Objective-C project with SonarCloud and https://docs.sonarcloud.io/getting-started/github/[GitHub Actions]:
+It is very easy to analyze a C, C++ and Objective-C project (and compute its test coverage) with SonarCloud and https://docs.sonarcloud.io/getting-started/github/[GitHub Actions]:
 
 . Create a `sonar-project.properties` file to store your configuration
 . In your `.github/workflows/build.yml` file:
+.. Make sure you have gcovr v5.0 (v5.1 is not supported, see https://jira.sonarsource.com/browse/SONAR-16269[SONAR-16269])
```

Comment:
This line will not be relevant anymore when this ticket is done. Maybe add a comment in the ticket to not forget to update this README?

Reply:
Added the comment.